src/Refactoring-Core/RBClass.class.st

```diff
@@ -160,6 +160,14 @@ RBClass >> isEmptyClass [
 					  self selectors isEmpty]]
 ]
 
+{ #category : 'testing' }
+RBClass >> isManifest [ 
+
```

Comment:
Can you add some tests for this because it gives me shivers :)
In particular because reading it is cumbersome (how superclass can work in none realClass)
I know how but reading it is strange.

Reply:
Thanks Stef; I updated the isManifest implementation, so now it doesn't check the realClass. I also included a new test when the superclass is nil, and the implementation handles this case.